### ISSUE TITLE
[Feature Request] Tooltips

### ISSUE THREAD

**Issue by LeeThompson** [benphelps/homepage#513]
### Description

A nice option would be to have css-style tooltips appear when hovering over an item.

This could use the `description`: entry or perhaps a new `tooltip:` field.    (If the former, there could be a directive in `settings` to hide description from the regular display.)

**Comment by JazzFisch** [comment]
@LeeThompson can you provide some more context?

If the tooltip displayed the description field there would be some duplication.  Are you thinking that if you supply a tooltip and not a description you could save some real estate in the widget and then see more information on hover over?

Thanks.

**Comment by LeeThompson** [comment]
Yes, to keep the dashboard compact but allow additional information if needed.

**Comment by JazzFisch** [comment]
Thanks for the additional context.
